File: Data Structures & Algorithms/is-anagram/submission-0.js
class Solution {
    /**
     * @param {string} s
     * @param {string} t
     * @return {boolean}
     */
    isAnagram(s, t) {
        if ( s.length != t.length) return false;

        let sortedS= s.split('').sort().join('');
        let sortedT= t.split('').sort().join('');

    return sortedS === sortedT;
    }
}
let obj = new Solution();
console.log(obj.isAnagram("silent","listen"))
console.log(obj.isAnagram("car","tar"))

File: Data Structures & Algorithms/last-stone-weight/submission-0.py
class Solution:
    def lastStoneWeight(self, stones: List[int]) -> int:
        if len(stones) == 0:
            return 0
        elif len(stones) == 1:
            return stones[0]

        stones.sort()
        y = stones.pop()
        x = stones.pop()

        if x != y:
            stones.append(y-x)

        return self.lastStoneWeight(stones)

        